testing/src/test/java/org/motechproject/nms/testing/it/flwUpdate/FrontLineWorkerUpdateImportServiceBundleIT.java

```diff
@@ -723,4 +744,40 @@ public void verifyFT561() throws InterruptedException, IOException {
         assertEquals("flw_FT_561.txt", csvAuditRecord.getFile());
     }
 
+    /**
+     * Method used to add Bookmark, Completion and Activity record with given contactNumber
+     */
+    private void createMaRecords(Long contactNumber) {
+
+        bookmarkDataService.create(new Bookmark(contactNumber.toString(), "1", "1", "1", new HashMap<String, Object>()));
+
+        CompletionRecord cr = new CompletionRecord(contactNumber, 35, false, 1);
+        completionRecordDataService.create(cr);
+//        String externalId, String courseName, String chapterName, String lessonName, DateTime startTime, DateTime completionTime, ActivityState.STARTED);
+        ActivityRecord ar = new ActivityRecord(contactNumber.toString(), "1", "1", "1", null,null , ActivityState.STARTED);
+        activityDataService.create(ar);
+    }
+
+    private void assertMaRecords(Long oldContactNumber, Long newContactNumber) {
```

Comment:
Added one negative case since for all negative code path irrespective of Bookmark/Completion/ActivityRecord Update, Flw update will be rejected.